src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -58,21 +67,26 @@ export const PersonalInfo: React.FC<IEditAuthorProps> = ({
     regionId: author?.region.id ?? null,
     cityId: author?.mainInstitution.city.id ?? null,
     bio: author?.bio ?? '',
-    email: author?.email ?? '',
-    socialNetwork: author?.socialNetworks ?? [null, null, null, null, null],
-    work: author?.mainInstitution.name ?? '',
+    publicEmail: author?.publicEmail ?? '',
+    socialNetworks: author?.socialNetworks ?? [null, null, null, null, null],
+    mainWorkingPlace: author?.mainInstitution.name ?? '',
   });
   const previousAuthorValues = usePrevious<INewAuthorValues>(newAuthorValues);
+  const [isLoading, setIsLoading] = useState(false);
 
+  const history = useHistory();
+  const authorities = useSelector(selectAuthorities);
+  const [boundGetUserAsyncAction] = useActions([getUserAsyncAction]);
+  const isAdmin = authorities.data?.includes('SET_IMPORTANCE');
```

Comment:
by a chance, may we have it already as a separate utility function?

Reply:
do you mean ?
```typescript 
  const authorities = useSelector(selectAuthorities);
  const isAdmin = authorities.data?.includes('SET_IMPORTANCE');
```

> by a chance, may we have it already as a separate utility function?

Reply:
either for part

```
authorities.data?.includes('SET_IMPORTANCE')
```
or for whole

```
useSelector(selectAuthorities).data?.includes('SET_IMPORTANCE')
```

Reply:
No, we don`t

> either for part
> 
> ```
> authorities.data?.includes('SET_IMPORTANCE')
> ```
> 
> or for whole
> 
> ```
> useSelector(selectAuthorities).data?.includes('SET_IMPORTANCE')
> ```

Reply:
may you plz introduce a helper and update/refactor only this occurrence(only this - by now)? I  can see [there are 12 locations](https://github.com/search?q=repo%3Aita-social-projects%2Fdokazovi-fe%20SET_IMPORTANCE&type=code) with such a check across our codebase - but sure, refactoring all them in scope of current ticket is suboptimal

Reply:
Okay, I will do it